### PR TITLE
Add more file type(s)

### PR DESCRIPTION
**Pull request by realeatham**

### CODE REVIEW

File: packages/shared/types/index.ts [hunk 1]
```diff
@@ -138,6 +138,8 @@ export enum FileType {
     WEBP = "webp",
     JPEG = "jpeg",
     SEVEN_Z = "7z",
+    TAR_GZ = "tar.gz", // Possibly Redundant?
```
Comment (Abhinav5383) [inline packages/shared/types/index.ts:141]: The way file types are resolved currently, no file will ever resolve to this type
so yeah this is redundant

Comment (realeatham) [reply packages/shared/types/index.ts:141]: I will remove it

Comment (realeatham) [reply packages/shared/types/index.ts]: done

**Review by Abhinav5383 — APPROVED** [review]
all good

**Comment by realeatham** [comment]
k i will merge

**Pull request merged**